core/src/test/scala/examples/DummyConsumer.scala

```diff
@@ -9,6 +9,8 @@ import org.apache.kafka.common.serialization.{ByteArrayDeserializer, StringDeser
 
 import scala.concurrent.Await
 import scala.util.Failure
+import com.softwaremill.react.kafka2._
+import scala.concurrent.duration._
 
 object Streams {
```

Comment:
This Streams object is used by both `DummyConsumer` and `DummyProducer`. Should it be in a separate file, or should all the three be in the same file?